dvc/stage.py

```diff
@@ -1043,13 +1050,9 @@ def status(self, check_updates=False):
         return {}
 
     def _already_cached(self):
-        return (
-            not self.changed_md5()
```

Comment:
This was not really needed, as all we care about is cmd its deps (and their hashes) and outs. Things like `always_run` are handled separately.